leonardo-example.conf

```diff
@@ -182,6 +182,16 @@ autoFreeze {
   autoFreezeCheckScheduler = 1 minutes
 }
 
+#Configs to auto delete after idle period
```

Comment:
we don't need to update this file anymore...don't think the content is kept up-to-date